### ISSUE TITLE
Abnormal circulating hydroxybutyric acid

### ISSUE THREAD

**Issue by JillMiller** [obophenotype/human-phenotype-ontology#6010]
**Preferred term label:** Abnormal circulating hydroxybutyric acid

**Synonyms**

**Definition (free text, please give PubMed ID)** Any deviation form the normal concentration of a hydroxybutiric acid in the blood circulation.

**Parent term (use hpo.jax.org/app)** Abnormal circulating metabolite concentration HP:0032180

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by JillMiller** [comment]
**Preferred term label:** Elevated circulating 2-Hydroxybutyric acid

**Synonyms**

**Definition (free text, please give PubMed ID)** 2-Hydroxybutyric acid concentration in the blood is increased above the normal range., PMID: 27208342

**Parent term (use hpo.jax.org/app)** Retrieving data. Wait a few seconds and try to cut or copy again.

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by JillMiller** [comment]
**Preferred term label:** Elevated circulating 3-hydroxybutyric acid

**Synonyms** Elevated β-Hydroxybutyrate; Elevated beta-hydroxybutyrate

**Definition (free text, please give PubMed ID)** Concentration of 3-hydroxybutyric acid in the blood is above the normal range., PMID: 28826372

**Parent term (use hpo.jax.org/app)** Abnormal circulating hydroxybutyric acid

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3058661/

**Comment by pnrobinson** [comment]
I think that only elevation is clinically relevant, we do not need the Reduced or Abnormal terms for these.

**Comment by pnrobinson** [comment]
added
